src/pkg/packager/prepare.go

```diff
@@ -120,36 +125,55 @@ func (p *Packager) FindImages() (imgMap map[string][]string, err error) {
 		if err != nil {
 			return nil, err
 		}
+		values, err := template.Generate(p.cfg)
+		if err != nil {
+			return nil, fmt.Errorf("unable to generate template values")
+		}
 
+		values.SetState(&types.ZarfState{
+			RegistryInfo: types.RegistryInfo{
+				Address: p.cfg.FindImagesOpts.RegistryURL,
+				// Adding a fake secret here so if a chart expects the registry secret, like the init package, it won't fail
+				Secret: "fake-zarf-secret",
```

Comment:
This feels kinda hacky, but I wanted find-images to work on the init package. I thought about making secret an option on find-images like registryurl but decided against as it doesn't actually pertain to the image like the former

Reply:
The `fillInEmpty*` functions in `cluster` could be useful here - they actually do not need the cluster struct and might work as receivers on types.ZarfState.

Reply:
Changed to be receivers on the respective objects